WIPv73Changelog.md

```diff
@@ -0,0 +1,799 @@
+  ```
+
+- `CheckoutSessionLineItemPriceDataRecurringParams.AggregateUsage`, `CheckoutSessionLineItemPriceDataRecurringParams.TrialPeriodDays`, and `CheckoutSessionLineItemPriceDataRecurringParams.UsageType`
+- `CheckoutSessionPaymentIntentDataParams.Params`, `CheckoutSessionSetupIntentDataParams.Params`, `CheckoutSessionSubscriptionDataParams.Params`. `Params` should only be embedded in root method struct, and has extraneous fields not applicable to child/sub structs.
+- `CheckoutSessionTotalDetailsBreakdownTax.TaxRate`. Use `CheckoutSessionTotalDetailsBreakdownTax.Rate`
+- `CheckoutSessionTotalDetailsBreakdownTax.Deleted`
+- `CustomerParams.Token`
+- `Discount` `APIResource` embed
+- `FilePurposeFoundersStockDocument` (`"founders_stock_document"` option for `File.Purpose`)
+- `InvoiceParams.Paid`. Use `invoice.status` to check for status. `invoice.status` is a read-only field.
+- `InvoiceParams.SubscriptionPlan` and `InvoiceParams.SubscriptionQuantity` (note: these would have been on `InvoiceUpcomingParams`)
+- `InvoiceListLinesParams.Customer` and `InvoiceListLinesParams.Subscription` (these are not available for Invoice `ListLines`, but are available for `List`)
+- `IssuingAuthorizationRequestHistoryViolatedAuthorizationControlEntity` and `IssuingAuthorizationRequestHistoryViolatedAuthorizationControlName` (unused enums)
+- `IssuingCardSpendingControlsParams.SpendingLimitsCurrency`. `issuing_card` has `currency`, and `issuing_card.spending_controls.spending_limits.amount` will use that currency
+- `IssuingDisputeEvidenceServiceNotAsDescribed.ProductDescription`, `IssuingDisputeEvidenceServiceNotAsDescribed.ProductType`, `IssuingDisputeEvidenceServiceNotAsDescribedParams.ProductDescription`, `IssuingDisputeEvidenceServiceNotAsDescribedParams.ProductType`, and `IssuingDisputeEvidenceServiceNotAsDescribedProductType`. `issuing_dispute.evidence.service_not_as_described` does not have `product_description` or `product_type`. `issuing_dispute.evidence.canceled` does.
+- `LineItemTax.TaxRate`. Use `LineItemTax.Rate` instead.
+- `LineItem.Deleted`
+- `LoginLink.RedirectURL`
+- `PaymentIntentOffSession` (unused enum)
+- `PaymentIntentConfirmParams.PaymentMethodTypes`
+- `PaymentMethodFPX.TransactionID`
+- `Payout.BankAccount` and `Payout.Card` (These fields were never populated, use `PayoutDestination.BankAccount` and `PayoutDestination.Card` instead)
+- `PlanParams.ProductID`. Use `PlanParams.Product.ID` instead.
+
+  ```go
+  // Before
+  params := &stripe.PlanParams{
+    ProductID: stripe.String("prod_12345abc"),
+  }
+
+  // After
+  params := &stripe.PlanParams{
+    Product: &stripe.PlanProductParams{
+      ID:                  stripe.String("plan_id"),
+    },
+  }
+  ```
+- Remove `Shipping` and `ShippingRate` properties from `CheckoutSession` resource. Please use `ShippingCost` and `ShippingDetails` properties instead.
+- Remove `DefaultCurrency` property from `Customer` resource. Please use `Currency` property instead.
```

Comment:
Should we set up a standard format for how we want to phrase removes (ie. when do we include remove prefix vs when do we not?)

Reply:
Yes, will do today